src/views/pages/agenda.js

```diff
@@ -91,12 +92,20 @@ function sessionCard(session, isFavorite) {
                             Text(rooms[session.attributes.room].name),
                         ])
                             .direction("vertical"),
-                        Actionable(
-                            Icon("local_fire_department")
-                                .color(isFavorite ? colors.LenraColors.yellowPulse : colors.Colors.black)
-                                .style(isFavorite ? "rounded" : "outlined")
-                        )
-                            .onPressed(listeners.toggleFavorite, { session: session.attributes.key }),
+                            Flex(
+                                [
+                                    Actionable(
+                                        Icon("local_fire_department")
+                                            .color(isFavorite ? colors.LenraColors.yellowPulse : colors.Colors.black)
+                                            .style(isFavorite ? "rounded" : "outlined")
+                                    ).onPressed(listeners.toggleFavorite, { session: session.attributes.key }),
+                                    Actionable(
+                                        Icon("forum")
+                                            .color(colors.Colors.black)
+                                            .style("rounded")
+                                    ).onPressed("@lenra:navTo", { path: openfeedbacks[session.attributes.key] }),
+                                ]
+                            ).direction("vertical"),
```

Comment:
I think that an horizontal flex would be better and you should add a space between the elements and reverse there order.

```suggestion
                            ).spacing(8),
```